tools/matchms/matchms_formatter.xml

```diff
@@ -52,7 +52,7 @@
 
     </inputs>
     <outputs>
-        <data label="${tool.name} (${method.selection}) on ${on_string}" name="output" format="tsv" />
+        <data label="${tool.name} (${method.selection}) on ${on_string}" name="output" format_source="scores" metadata_source="scores" />
```

Comment:
What sort of metadata are we looking to source from the input file here?

Reply:
You are right, it doesn't make sense in tabular type of data (fixed in 45ab1df).